openhands/integrations/templates/resolver/github/pr_update_prompt.j2

```diff
@@ -1,25 +1,3 @@
-You are checked out to branch {{ branch_name }}, which has an open PR #{{ pr_number }}.
-A comment on the PR (below) has been addressed to you. Do NOT respond to this comment via the GitHub API.
+# PR Comment
```

Comment:
maybe ditch this heading? this will be the first user message right?

Reply:
oh do we need a frontend change too?

Reply:
Yeah I was thinking in a separate PR but can do it here as well